src/browser/components/TabBar.jsx

```diff
@@ -53,17 +55,19 @@ function TabBar(props) {
         { badgeDiv }
       </NavItem>);
   });
-  tabs.push(
-    <NavItem
-      className='TabBar-addServerButton'
-      key='addServerButton'
-      id='addServerButton'
-      eventKey='addServerButton'
-      title='Add new server'
-    >
-      <Glyphicon glyph='plus'/>
-    </NavItem>
-  );
+  if (AppConfig.data.enableMultiTeams === true) {
```

Comment:
Probably `enableServerManagement` should be referred here instead of `enableMultiTeams`.

And please make components stateless as well as possible. So the value should be passed via props. 
For example, add `showAddServerButton` prop to `TabBar.jsx` and `MainPage.jsx`, then pass the parameter at `ReactDOM.render()` of `index.jsx`.

Reply:
@yuya-oc refactored to keep `TabBar.jsx` stateless. The right condition is to hide the plus is when `enableMultiTeams= true` since that's when you can add more teams.